homeworks/HW_05/task3_1_while.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const constNumber = 12;
+let count = 0;
+let multiplyFunction = '';
+
+while (count < 10) {
+    count++;
+    multiplyFunction = constNumber * count;
+    console.log(`${constNumber} x ${count} = ${multiplyFunction}`);
```

Comment:
Можна і так, але найлегше було б просто не виводити результат множення до окремої змінної
```suggestion
    console.log(`${constNumber} x ${count} = ${constNumber * count}`);
```